main.tf

```diff
@@ -71,6 +71,7 @@ resource "google_compute_instance" "vm_instance" {
   depends_on                = [google_project_service.compute_api]
   lifecycle {
     ignore_changes = [
+      attached_disks,
```

Comment:
Hey. Adding just the attribute name in `ignore_change[]` won't work. We also need to add an initial value for the attribute in the resource itself. Please refer to [this commit](https://github.com/airasia/terraform-google-vm_instance/commit/fe31d1ec3069cc88fc236737d38bceff7226767b) where we did similar thing for `metadata[windows-keys]`